source/models/working-with-relationships.md

```diff
@@ -0,0 +1,116 @@
+### Creating Records
+
+Let's assume that we have a `post` and a `comment` model, which are related to each other as follows:
+
+```app/models/post.js
+export default DS.Model.extend({
+  comments: DS.hasMany('comment')
+});
+```
+
+```app/models/comment.js
+export default DS.Model.extend({
+  post: DS.belongsTo('post')
+});
+```
+
+When a user comments on a post, we need to create a relationship between the two records. We can simply set the `belongsTo` relationship in our new comment:
+
+```javascript
+let post = this.store.peekRecord('post', 1);
+let comment = this.store.createRecord('comment', {
+  // comment data
+  post: post
+});
+comment.save();
+```
+
+This will create a new `comment` record and save it to the server. Ember Data will also update the post to include our newly created comment in its `comments` relationship. We could have also linked the two records together by updating the post's `hasMany` relationship:
+
+```javascript
+let post = this.store.peekRecord('post', 1);
+let comment = this.store.createRecord('comment', {
+  // comment data
+});
+post.get('comments').pushObject(comment);
+comment.save();
+```
+
+In this case the new comment's `belongsTo` relationship will be set to the parent post.
+
+### Updating Existing Records
+
+Sometimes we want to set relationships on already existing records. We can simply set a `belongsTo` relationship:
+
+```javascript
+let post = this.store.peekRecord('post', 1);
+let comment = this.store.peekRecord('comment', 1);
+comment.set('post', post);
+comment.save();
+```
+
+We can also update the `hasMany` relationship by pushing a record into the relationship:
+
+```javascript
+let post = this.store.peekRecord('post', 1);
+let comment = this.store.peekRecord('comment', 1);
+post.get('comments').pushObject(comment);
+post.save();
+```
+
+### Removing Relationships
+
+To remove a `belongsTo` relationship, we can just set it to `null`, which will also remove it from the inverse:
+
+```javascript
+let comment = this.store.peekRecord('comment', 1);
+comment.set('post', null);
+comment.save();
+```
+
+It is also possible to remove a record from a `hasMany` relationship:
+
+```javascript
+let post = this.store.peekRecord('post', 1);
+let comment = this.store.peekRecord('comment', 1);
+post.get('comments').removeObject(comment);
+post.save();
+```
+
+As in the earlier examples, the comment's `belongsTo` relationship will also be cleared by Ember Data.
+
+### Relationships as Promises
+
+While working with relationships it is important to remember that they return promises. In the examples above we have assumed that the records we were working on were already loaded into the store, but this may not always be the case.
```

Comment:
In the above examples, we aren't actually working with the returned data, so that they returned promises isn't really relevant. I think removing the last sentence would avoid confusion about when the user needs to be concerned about the asynchronous nature of the relationship.